### ISSUE TITLE
Improve position of outgoing/incoming demand button

### ISSUE THREAD

**Issue by aubergine10** [CitiesSkylinesMods/TMPE#832]
This is the button that appears on PT info view when Advanced Parking AI is active: https://github.com/CitiesSkylinesMods/TMPE/wiki/Public-Transport-Info-View

Allystrya 12 hours ago:

> Is there a way to move the outgoing/incoming demand button? I have tried different mouse and keyboard w/mouse combinations but nothing seems to work and I didn't see anything in the Wiki.
>  
> I didn't even know where it was until here recently when I was sent a screenshot but because of the resolution I have to use it sits right behind the transport window and I can barely see the top of it. I can tell I can click it because of the buildings that change however i am not sure if I am on outgoing on incoming because I can't see the rest :)
>  
> If there isn't a way to move it, then is there any way to tell which demand channel I am on?

**Comment by krzychu124** [comment]
Btw, I think there is no info in wiki about **Parking space demand** view when you select _Traffic info view_ while TM:PE menu is visible

**Comment by aubergine10** [comment]
The _what_ view?

These are what I currently have on the info views:

* https://github.com/CitiesSkylinesMods/TMPE/wiki/Public-Transport-Info-View
* https://github.com/CitiesSkylinesMods/TMPE/wiki/Traffic-Info-View
* https://github.com/CitiesSkylinesMods/TMPE/wiki/Traffic-Routes-Info-View

Regarding traffic info view, I thought it always showed where it's difficult to find parking regardless of TMPE menu state?

I will test next time in-game to see if having TMPE toolbar open changes anything.

**Comment by krzychu124** [comment]
I'm blind, lol... 
Menu state doesn't do anything besides showing actual demand numbers when you turn on building debug info overlay

**Comment by aubergine10** [comment]
Just tested, couldn't see any difference between TMPE menu being open or not.

However, I also didn't notice any building colors being applied so maybe something is broken? Or maybe I provided enough parking? I'll see if it changes as my city grows.

**Comment by krzychu124** [comment]
Yeah, public transport view seem to be working, but parking demand not at all.
Values of parking demand (debug view) doesn't seem to be changing, even if I disable parking roads in whole/half city... 
It looks like they walk by foot instead of trying to use cars xD

**Comment by aubergine10** [comment]
From vague memory of a discussion with Victor some years ago, the parking thing is based on Parking AI reaching destination and then having to try find somewhere to park - each subsequent attempt will bump the value for parking issues on the building the cim is trying to get to.

**Comment by krzychu124** [comment]
Yup, exactly that's in the code, but I have to figure out how to force cims to fail while they're searching for parking space 😄
